### PR TITLE
Add Change columnsize

### PR DESCRIPTION
**Pull request by MarcRaeder**
Add Change columnsize feature

### CODE REVIEW

File: table.js [hunk 1]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+  var rightColumnWidth = columns[borderNum + 1].offsetWidth;
+
+  if (borderNumber > 0) {
+    for (let i = 0; i < borderNumber; i++) {
+      previousColumsWidth += columns[i].offsetWidth;
+    }
+  }
+
+  function mousemove(event) {
+    var mouseend = event.clientX;
```
Comment (stefanluth) [inline table.js]: camelCase :)

File: table.js [hunk 2]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+    var mouseDifference = mousestart - mouseend;
+    var leftborder = leftColumnWidth < minWidth && event.clientX >= (previousColumsWidth + minWidth);
```
Comment (stefanluth) [inline table.js]: camelCase :)

File: table.js [hunk 3]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+    var outOfBorder = event.clientX < (previousColumsWidth + minWidth) || event.clientX > (previousColumsWidth + leftColumnWidth + (rightColumnWidth - minWidth));
+    var rightborder = rightColumnWidth < minWidth && event.clientX <= (previousColumsWidth + leftColumnWidth + minWidth);
```
Comment (stefanluth) [inline table.js]: camelCase :)

File: table.js [hunk 4]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+      border[i].style.left = `${border[i - 1].offsetLeft + columns[i].offsetWidth}px`;
+    }
+  }
+}
+
+document.getElementsByTagName("BODY")[0].onresize = () => resize();
+
+function dragstart(event, borderNumber) {
+  const minWidth = 100;
+  const mousestart = event.clientX;
```
Comment (stefanluth) [inline table.js]: camelCase :)

File: table.js [hunk 5]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+    }
+    else {
+      border[i].style.left = `${border[i - 1].offsetLeft + columns[i].offsetWidth}px`;
+    }
+  }
+}
+
+document.getElementsByTagName("BODY")[0].onresize = () => resize();
+
+function dragstart(event, borderNumber) {
```
Comment (stefanluth) [inline table.js]: camelCase :)

File: table.js [hunk 6]
```diff
@@ -9,3 +13,74 @@ function showHideData(columnClass) {
...
+  var rightColumn = columns[borderNum + 1];
+  var rightColumnWidth = columns[borderNum + 1].offsetWidth;
+
+  if (borderNumber > 0) {
+    for (let i = 0; i < borderNumber; i++) {
+      previousColumsWidth += columns[i].offsetWidth;
+    }
+  }
+
+  function mousemove(event) {
```
Comment (stefanluth) [inline table.js]: camelCase :)

**Review by stefanluth — CHANGES_REQUESTED** [review]

**Review by matthiasdanne — CHANGES_REQUESTED** [review]
What Stefan said

**Comment by DevBastiS** [comment]
What Matthias said

File: table.css [hunk 7]
```diff
@@ -16,11 +22,21 @@
...
   border: 1px solid black;
+  border-left: 1px hidden black;
 }
 
 .table__content-row-cell {
   border: 1px solid black;
   min-width: 100px;
   border-left: 1px hidden black;
   background-color: burlywood;
+
```
Comment (stefanluth) [inline table.css:33]:
```suggestion
```

**Review by stefanluth — APPROVED** [review]

**Review by DevBastiS — APPROVED** [review]

**Pull request merged**